src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/CronScheduleUtil.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using NCrontab;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Services
+{
+    public static class CronScheduleUtil
+    {
+        public static DateTime GetNextScheduledPublishingTime()
+        {
+            var publishReleasesCronSchedule = Environment.GetEnvironmentVariable("PublishReleaseContentCronSchedule");
```

Comment:
I'm guessing we haven't done this in other places, but I think it could be a good idea to aggregate environment variable configuration into a single config class. That way we'd be able to strongly type what the environment variables should be and not lose track of them in the future.

Reply:
We have done this in one place and attempted to do it before elsewhere. I raised this new issue because I think we need to do it consistently.

https://dfedigital.atlassian.net/browse/EES-918

Reply:
:+1: